### DISCUSSION TITLE
How to remove the Button used to `open & close Navigation`

### DISCUSSION THREAD

**Discussion by Jhonfunk** [bdlukaa/fluent_ui#971] [Q&A] [answered]
![ 2023-11-21 165100](https://github.com/bdlukaa/fluent_ui/assets/78594391/54567584-ba28-4fdd-98f2-ce6d4dfda90d)

**Comment by bdlukaa** [comment, marked as answer]
On compact mode, you can not remove the toggle button. Could you file an issue about this?

**Comment by Jhonfunk** [comment]
#973
